hiss/user/admin.py

```diff
@@ -1,10 +1,44 @@
 from django.contrib import admin
+from django.utils import timezone
+
 from user.models import User
 
 
-# Register your models here.
+def check_in(modeladmin, request, queryset):  #TO-DO: Needs to be tested!!!
```

Comment:
```suggestion
def check_in(modeladmin, request, queryset) -> None:
```

Please add a test for this Django admin action. For more information, see this [StackOverflow question](https://stackoverflow.com/questions/29026779/testing-custom-admin-actions-in-django). In addition, please add a docstring indicating what this function is supposed to do.